src/derivatives/atlas.md

```diff
@@ -0,0 +1,961 @@
+as BIDS-Derivatives.
+These outcomes typically involve quantitative maps, feature maps, parcellations,
+segmentations, and other knowledge annotations such as landmarks defined with
+respect to individual or template brains that are supported by spaces
+such as surfaces and regular grids (images).
+
+For derivatives with atlases in their provenance corresponding to individual subjects,
+the organization follows the standards for BIDS raw and derivatives.
+The following entities MAY be employed to specify template- and atlas-derived results:
+
+-    [`space-<label>`](../glossary.md#space-entities) is REQUIRED to disambiguate derivatives defined with
+     respect to different [coordinate systems](../appendices/coordinate-systems.md), following the general
+     BIDS-Derivatives specifications.
+-    [`cohort-<label>`](../glossary.md#cohort-entities) is REQUIRED to disambiguate derivatives defined with
+     respect to different cohort instances of a single [space (coordinate system)](../appendices/coordinate-systems.md).
+-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to encode files pertaining
+     or derived from the atlas identified by the entity's label.
+-    [`seg-<label>`](../glossary.md#segmentation-entities) is REQUIRED when a single atlas has several different
+     realizations (for instance, segmentations and parcellations created with different criteria) that
+     need disambiguation.
+-    [`scale-<label>`](../glossary.md#scale-entities) is REQUIRED to disambiguate different atlas 'scales',
+     when the atlas has more than one 'brain unit' resolutions, typically relating to the area covered
+     by regions.
+
+The general filename pattern for subject derivatives with templates and atlases in their provenance
+follows the general BIDS-Derivatives pattern:
+
+```Text
+<pipeline_name>/
+    sub-<label>/
+        <datatype>/
+            <source_entities>[_space-<space>][_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
+```
+
+[`atlas-<label>`](../glossary.md#atlas-entities), [`seg-<label>`](../glossary.md#segmentation-entities),
+and [`scale-<label>`](../glossary.md#scale-entities) are discussed later in section
+[Filenames of derivatives with atlases in their provenance](#filenames-of-derivatives-with-atlases-in-their-provenance).
+
+For derivatives of template- and altas-generating pipelines, which typically aggregate
+several sessions and/or subjects, the derivatives-specific
+[`tpl-<label>` entity](../glossary.md#template-entities) is dual in terms of usage to BIDS raw's
+[`sub-<label>`](../glossary.md#subject-entities), and MAY be employed as follows:
+
+```Text
+<pipeline_name>/
+    tpl-<label>/
+        [cohort-<label>/]
+           [<datatype>/]
+               <source_entities>[_cohort-<label>][_space-<space>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Comment:
We should explicitly state (and add a schema check) that `tpl` and `space` are mutually exclusive.

```suggestion
    tpl-<label>/
        [cohort-<label>/]
           [<datatype>/]
               tpl-<label>_<source_entities>[_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Reply:
Should we also remove `<source_entities>` here?

```suggestion
    tpl-<label>/
        [cohort-<label>/]
           [<datatype>/]
               tpl-<label>[_cohort-<label>][_space-<space>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Reply:
If the original data is from a functional localizer, would you need `task-<label>`, or are we sufficiently removed that information should go in some other entity?